cmake.yaml

```
@@ -37,6 +37,7 @@ pipeline:
         --datadir=/share/cmake \
         --docdir=/share/doc/cmake \
         --system-libs \
+        --no-system-cppdap \
```

```suggestion
        --no-system-cppdap \
```

It would be good to add a comment explaining why the bundled dependencies are used (since they themselves depend on CMake, creating a cycle).

---

See [f10be8b](https://github.com/wolfi-dev/os/pull/3810/commits/f10be8b1a40fd146d1839033e216fb51ba1c049d)